src/scanner.coffee.md

```diff
@@ -139,14 +139,15 @@ of cells or null if the scanner is exhausted.
         return callback() if response and response.statusCode is 204
         return callback err if err
         cells = []
+        encoding = if @options.encoding is 'undefined' then @options.encoding else @client.options.encoding
```

Comment:
yes, this would be enough. I just don't understand why the Scanner.init accepts an option.encoding
`encoding = if @options.encoding is 'undefined' then @options.encoding else @client.options.encoding`
If this is not necessary then a default in client.option would be surely enough